common/common.js

```diff
@@ -19,6 +19,8 @@ configs = new Configs({
 	ieargs       : '',
 	forceielist  : '',
 	disableForce : false,
+	exclusionlist  : '',
```

Comment:
A camelCase name looks better for this option. Moreover, `sitesOpenedBySelf` or something look more meaningful instead of the term "exclusion".